### PR TITLE
refactor: use null instead of {}

### PR DESCRIPTION
**Pull request by sethidden**
In Checkout.vue and Billing.vue there was a case where `getSomeData() || {}` was used. This was probably done to avoid worrying if something is null or not and avoiding the "x is null, trying to access property abc" error.

The problem with that is that it also causes problems with typing, as you'd get errors like "X is not assignable to {}"

### CODE REVIEW

File: packages/theme/helpers/checkout/address.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import type { CustomerAddress, CartAddressInterface } from '~/modules/GraphQL/types';
```
Comment (sethidden) [inline packages/theme/helpers/checkout/address.ts:1]: (file was moved from address.js)

File: packages/theme/helpers/checkout/address.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  street: string;
+  apartment: string;
+  city: string;
+  region: string;
+  country_code: string;
+  postcode: string;
+  telephone: string;
+}
+
+export const getInitialCheckoutAddressForm = () : CheckoutAddressForm => ({
```
Comment (sethidden) [inline packages/theme/helpers/checkout/address.ts:45]: Thanks to this, the {} can be avoided

File: packages/theme/modules/checkout/getters/cartGetters.ts [hunk 3]
```diff
@@ -140,9 +140,6 @@ export const getTotalItems = (cart: Cart): number => {
 
 export const getConfiguredVariant = (product: ConfigurableCartItem): ProductInterface | null => product?.configured_variant || null;
 
-// eslint-disable-next-line import/no-named-as-default-member
-export const getFormattedPrice = (price: number) => getFormattedPrice(price);
```
Comment (sethidden) [inline packages/theme/modules/checkout/getters/cartGetters.ts:144]: removed weird self-referencing getter. it's not used anywhere

File: packages/theme/helpers/checkout/address.ts (hunk 1 above)
Comment (Frodigo) [reply packages/theme/helpers/checkout/address.ts:1]: Maybe the file should be located in modules/checkout/helpers?

**Review by Frodigo — APPROVED** [review]

**Review by Frodigo — APPROVED** [review]

**Pull request merged**